www/js/modules/charts/controllers/HistogramChartController.js

```diff
@@ -157,7 +160,7 @@ export default class HistogramChartController {
 
         if (!this.$tooltip) {
 
-            this.$tooltip = this.histogramChartModel.$parent
+            this.$tooltip = d3.select('html') //this.histogramChartModel.$parent
                 .append('div')
```

Comment:
@tniswong you were attaching the tooltip to the histogram chart html element.  However, the coordinates are pulled from the page as a whole.  So when plotting the tooltip it was using incorrect coordinates.  Simple fix is to just select the root html element so that the coordinates will be correct.